core/src/tile/tileManager.cpp

```diff
@@ -237,6 +238,9 @@ void TileManager::updateTileSet(TileSet& _tileSet, const ViewState& _view,
                 entry.task = _tileSet.source->createTask(visTileId);
                 enqueueTask(_tileSet, visTileId, _view);
                 m_tilesInProgress++;
+            } else if (!entry.needsLoading() && !entry.isReady()) {
+                // tile not generated yet!
+                m_tilesInGeneration++;
             }
```

Comment:
^ Basically merge `m_tilesGenerating` in `m_tilesInProgress`. I am fine with that, can add some code comments along with it to mention what we mean by `InProgress`.